route/build/src/read_restart.f90

```diff
@@ -239,6 +249,61 @@ SUBROUTINE read_IRFbas_state(ierr, message1)
 
   END SUBROUTINE read_IRFbas_state
 
+  SUBROUTINE read_bas_tracer_state(ierr, message1)
```

Comment:
Seems like this should have at least some description of what it does.

Reply:
subroutine name is changed to `read_bas_solute_state`.   Wondering how much descriptions are needed. This and a bunch of other same routines are just reading state (or restart file).

Reply:
Just do a general description before the specific subroutines. So only done once...